src/backend/joanie/core/models/products.py

```diff
@@ -563,3 +540,74 @@ def set(self):
                 self.state = enums.ENROLLMENT_STATE_FAILED
             else:
                 self.state = enums.ENROLLMENT_STATE_SET
+
+
+def on_enrollment_orders_linked(action, instance, pk_set, **kwargs):
+    """Process some checks before link orders to an enrollment."""
```

Comment:
these checks are normally done in the model's validate_* methods or in the form if there is any?